src/common/options/ceph-exporter.yaml.in

```diff
@@ -52,3 +52,12 @@ options:
   - ceph-exporter
   flags:
   - runtime
+- name: exporter_get_labeled_counters
+  type: bool
+  level: advanced
+  desc: If true will fetch labeled perfcounters from counter dump command
+  default: false
+  services:
+  - ceph-exporter
+  flags:
+  - runtime
```

Comment:
Can't we just run the command if it exists? We are probably giving the user too much options to think about.